src/coreclr/jit/gentree.cpp

```diff
@@ -27476,6 +27476,7 @@ bool GenTreeHWIntrinsic::OperIsMemoryLoad(GenTree** pAddr) const
                 addr = Op(3);
                 break;
 
+            case NI_Sve_GatherVectorWithByteOffsets:
```

Comment:
remind me, but why this is not with other gathers on line 27517?

Reply:
`NI_Sve_GatherVectorWithByteOffsets` only comes in one version (scalar address with vector offsets). So it doesn't need to do the `if (varTypeIsSIMD(addr->gtType))` check as that will always be false.